src/native_main.nim

```diff
@@ -104,9 +104,11 @@ proc handleMessage(msg: MessageRecv): string =
                 reply.code = some(1)
 
         of "run":
-            var command = msg.command.get()
             when defined(windows):
```

Comment:
`when` is a bit like an `#ifdef` so you can "declare things twice". With these changes `command` is immutable so in theory it'll be a bit quicker.